torch/fft/__init__.py

```diff
@@ -549,3 +555,177 @@
     tensor([ 2.0000+-0.0000j, -0.5000-0.6882j, -0.5000-0.1625j, -0.5000+0.1625j,
         -0.5000+0.6882j])
 """)
+
+fftfreq = _add_docstr(_fft.fft_fftfreq, r"""
+fftfreq(n, d=1.0, *, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
+
+Returns Discrete Fourier Transform sample frequencies for signal size :attr:`n`.
+
+Note:
+
+    By convention, the FFT returns positive frequency terms first, followed by
+    the negative frequencies in reverse order, so that ``f[-i]`` in python
+    gives the negative frequency terms. For an FFT of length :attr:`n` and
+    with inputs spaced in length unit :attr:`d`, the frequencies are::
+
+        f = [0, 1, ..., (n - 1) // 2, -(n // 2), ..., -1] / (d * n)
+
+Note:
+    For even lengths, the Nyquist frequency at ``f[n/2]`` can be thought of as
+    either negative or positive. :func:`~torch.fft.fftfreq` follows NumPy's
+    convention of taking it to be negative.
+
+
+Args:
+    n (int): the FFT length
+    d (float, optional): The sampling length scale.
+        The unit length (or time) between individual samples in the FFT input.
+        If given, the frequencies are given in physical units of 1/unit.
+    {dtype}
+    {layout}
+    {device}
+    {requires_grad}
+
+Example:
+
+    >>> import torch.fft
+    >>> torch.fft.fftfreq(5)
+    tensor([ 0.0000,  0.2000,  0.4000, -0.4000, -0.2000])
+
+    In this example, the Nyquist frequency at ``f[2]`` is given as negative:
```

Comment:
This suggests that the Nyquist frequency may sometimes be positive here, but per the previous note it will always be negative.

Reply:
Changed "In this example" to "For even lengths".